src/test/java/org/ethereum/beacon/discovery/NodeRecordTest.java

```diff
@@ -114,4 +115,50 @@ public void testSignature() throws Exception {
     }
     assertTrue(exceptionThrown.get());
   }
+
+  @Test
+  public void shouldNotIncludePaddingInBase64() {
+    final int port = 30303;
+    final Bytes ip = Bytes.fromHexString("0x7F000001");
+    final Bytes nodeId =
+        Bytes.fromHexString("a448f24c6d18e575453db13171562b71999873db5b286df957af199ec94617f7");
+    final Bytes privateKey =
+        Bytes.fromHexString("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291");
+    final int seq = 1;
+
+    NodeRecord nodeRecord =
+        new NodeRecordFactory(new IdentitySchemaV4Interpreter())
+            .createFromValues(
+                UInt64.valueOf(seq),
+                Pair.with(EnrField.ID, IdentitySchema.V4),
+                Pair.with(EnrFieldV4.IP_V4, ip),
+                Pair.with(EnrFieldV4.UDP_V4, port),
+                Pair.with(
+                    EnrFieldV4.PKEY_SECP256K1, Functions.derivePublicKeyFromPrivate(privateKey)));
+    nodeRecord.sign(privateKey);
+    assertEquals(nodeId, nodeRecord.getNodeId());
+    final Bytes bytes = nodeRecord.serialize();
+    System.out.println(bytes);
```

Comment:
Stray `println`

Reply:
There's always one that sneaks through...